frontend/src/components/notFound/NotFound.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC } from 'react';
+import { Button } from 'react-bootstrap';
```

Comment:
Check out the naming principle in the project.
The same for the style file naming.